modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -1269,6 +1315,32 @@ void cv::gimpl::GFluidExecutable::run(std::vector<InObj>  &&input_objs,
     }
 }
 
+cv::gimpl::GParallelFluidExecutable::GParallelFluidExecutable(const ade::Graph &g,
+                 const FluidGraphInputData    &graph_data,
+                 const std::vector<GFluidOutputRois>        &parallelOutputRois)
+{
+    for (auto&& rois : parallelOutputRois){
+        tiles.emplace_back(g, graph_data, rois.rois);
+    }
+//    1. split the output images into tiles and pass each of them into separate GFluidExecutable obj
+
+}
+
+
+void cv::gimpl::GParallelFluidExecutable::reshape(ade::Graph&, const GCompileArgs& )
+{
+    GAPI_Assert(false && "Not Implemented;");
```

Comment:
Actually `reshape` can be easily implemented for `GParallelFluidExecutable`, that's outside of the scope of this review?

Reply:
you are right - not now.